package-lock.json

```diff
@@ -4587,12 +4609,14 @@
         "wrappy": {
           "version": "1.0.2",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
         },
         "yallist": {
           "version": "3.0.2",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
@forresttanaka and I have been noticing that `"optional": true` is added sometimes when new packages are installed and it looks like it is a known bug: https://github.com/npm/npm/issues/20926#issuecomment-397372131. We've been removing them because they don't seem important and we're trying to reduce changes in package-lock. I think maybe if you run `npm install` again they might be removed.

Reply:
@emro I threw the kitchen-sink at it and optional:true always stayed. I think it is due to an update in npm (https://stackoverflow.com/a/55373854/178550.) I can manually remove it if desired.

Reply:
OK, it looks like this is necessary, we should keep it.